Omise.Examples/Examples/Charges.cs

```diff
@@ -71,7 +71,7 @@ public async Task Create__Create_With_Customer()
         public async Task Capture__Capture()
         {
             var charge = RetrieveUncapturedCharge();
-            charge = await Client.Charges.Capture(charge.Id);
```

Comment:
@AnasNaouchi can we keep the original capture example while also adding the new capture-with-amount example?

Reply:
Yes we can